R/pivot_forecasts_wider.R

```diff
@@ -0,0 +1,57 @@
+#' Return a data frame of forecasts with predictive quantiles in columns.
+#'
+#' @param data forecasts data frame from load_forecast(). It has columns model, 
+#' forecast_date,location, target, type, quantile, value, horizon and 
+#' target_end_date.
+#' @param quantiles vector of quantiles to return for plotting
+#' Defaults to c(0.025, 0.1, 0.25, 0.5, 0.75, 0.9, 0.975). 
+#' Note that point forecasts and median quantiles do not necessarily align. 
+#' Throws an error if quantile levels are not in the forecast data.
+#'
+#' @return data frame with columns model, forecast_date, location, inc_cum, death_case, 
+#' horizon, type, point, target_end_date, point_forecast_type, Prediction Interval,
+#' upper, lower.
+#' 
+#' @export
+#' 
+pivot_forecasts_wider <- function(data, 
+                                  quantiles = c(0.025, 0.1, 0.25, 0.5, 0.75, 0.9, 0.975)){
```

Comment:
Does `quantiles` belong as an argument to this function?  If so, I'd rather it default to include all, e.g. via `sort(unique(data$quantile))`